### PR TITLE
Add subcalls information to referendums

### PR DESCRIPTION
**Pull request by ishrak-imam**
### Description
Added the subcalls information to referendums.

### CODE REVIEW

File: src/api/hooks/useFormatBalance.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {Compact} from '@polkadot/types';
 import type BN from 'bn.js';
 // eslint-disable-next-line no-restricted-imports
-import {formatBalance} from '@polkadot/util';
+import {formatBalance as formatBalancePolkaUtil} from '@polkadot/util';
```
Comment (ishrak-imam) [inline src/api/hooks/useFormatBalance.ts:4]: Just to fix eslint warning about having variable with same name in the upper scope.

File: src/ui/components/ProposalCallInfo.tsx [hunk 2]
```diff
@@ -1,25 +1,78 @@
...
 import {Padder} from '@ui/components/Padder';
-import type {DemocracyProposal, DemocracyReferendum} from 'src/api/hooks/useDemocracy';
+import type {DemocracyProposal, DemocracyReferendum, ProposalSubCall} from 'src/api/hooks/useDemocracy';
+import {standardPadding} from '@ui/styles';
 
 type ProposalInfoProps = {
   proposal: DemocracyProposal | DemocracyReferendum;
 };
 
 export function ProposalCallInfo({proposal}: ProposalInfoProps) {
```
Comment (ishrak-imam) [inline src/ui/components/ProposalCallInfo.tsx:12]: @Ekaanth 
You can use this component to display the call params of the motion proposals.

Comment (Ekaanth) [reply src/ui/components/ProposalCallInfo.tsx:12]: Yes, @ishrak-imam I'm using this component for proposals.

**Review by Ekaanth — APPROVED** [review]
LGTM!!

**Pull request merged**